assets/css/styles.css

```diff
@@ -2356,8 +2356,10 @@ body.no-header-wrapper {
     -moz-background-clip: padding;
     -webkit-background-clip: padding-box;
     background-clip: padding-box;
-    width: 100%;
-    padding: 0;
+    /* width: 100%; */
+    min-width: 90% !important;
+    padding-bottom: 90px;
```

Comment:
Please remove this and see if it's still ok. Should be. Seems quite big change and uneccessary. In as much as possible don't edit the styles already there